src/details-element-polyfill/polyfill.coffee.erb

```diff
@@ -56,11 +56,37 @@ polyfillProperties = ->
           removeAttribute.call(this, name)
 
 polyfillToggle = ->
-  onTogglingClick (element) ->
+  onTogglingTrigger (element) ->
+    summary = element.querySelector "summary"
     if element.hasAttribute("open")
       element.removeAttribute("open")
+      summary.setAttribute("aria-expanded", false)
     else
       element.setAttribute("open", "")
+      summary.setAttribute("aria-expanded", true)
+
+polyfillFocusAndARIA = ->
+  summaries = document.querySelectorAll("summary")
+  for summary in summaries
+    addAttributesForSummary(summary)
+  if MutationObserver?
+    observer = new MutationObserver (mutations) ->
+      for {addedNodes} in mutations
+        for target in addedNodes
+          if target.tagName is "DETAILS"
+            summary = target.querySelector "summary"
+            addAttributesForSummary(summary) if summary
+    observer.observe(document.documentElement, attributes: true, subtree: true, childList: true)
+  else
+    document.addEventListener "DOMNodeInserted", (event) ->
+      if event.target.tagName is "SUMMARY"
+        addAttributesForSummary event.target
```

Comment:
Ideally, we'd only create a single `MutationObserver` (or `DOMNodeInserted` listener). Think this could be combined with the one in `polyfillToggleEvent`?

Reply:
`polyfillToggleEvent` only get called when `support.element and not support.toggleEvent` while `polyfillFocusAndARIA` only get called when `not support.element`, so only one `MutationObserver` will ever get created. Do you think it's worth combining them still? The conditions make it a bit awkward when I tried. 🤔